### PR TITLE
fix hmr: btoa unicode support

### PR DESCRIPTION
**Pull request by boomfly**
When module that need to replaced have unicode symbols hrm client package throws error:

Снимок экрана 2020-10-29 в 14 54 08

This break replacement and to see your changes you need to reload full page.

### CODE REVIEW

File: packages/hot-module-replacement/client.js [hunk 1]
```diff
@@ -179,8 +179,21 @@ function walkTree(pathParts, tree) {
...
+const btoa = window.btoa
+const atob = window.atob
+
+// Replace with unicode support
+window.btoa = function utoa(data) {
+  return btoa(unescape(encodeURIComponent(data)));
+}
+
+window.atob = function atou(data) {
+  return decodeURIComponent(escape(atob(b64)));
```
Comment (namenotrequired) [inline packages/hot-module-replacement/client.js:192]: I cannot test at the moment, but at a glance; was `b64` meant to be `data`?

Comment (boomfly) [reply packages/hot-module-replacement/client.js]: Yes, my bad) but i don't now how it working for me.

Thanks for review. I remove this extra method. This is enough:
```javascript
function utoa(data) {
  return btoa(unescape(encodeURIComponent(data)));
}
```

**Comment by StorytellerCZ** [comment]
@boomfly Thank you for the fix! Please sign CLA so that we can consider it.

**Comment by boomfly** [comment]
@StorytellerCZ, ok. Done. i've already sign before. I need to sign on every PR?

**Comment by zodern** [comment]
@boomfly thanks for fixing this. Could you please create a new PR based off off the `hot-module-reload` branch?

**Comment by boomfly** [comment]
@zodern, done PR #11228

**Pull request closed without merging**